src/condor_utils/param_info.in

```diff
@@ -6802,6 +6802,7 @@ default: @end
    COLLECTOR$(1).COLLECTOR_NAME=Collector$(1)
    COLLECTOR$(1).CONDOR_VIEW_HOST=$(TOP_COLLECTOR_HOST)
    COLLECTOR$(1).CCB_RECONNECT_FILE=$(SPOOL)/Collector$(1).ccb_reconnect
+   COLLECTOR$(1).COLLECTOR_LOG=$(LOG)/Collector$(1)Log
```

Comment:
unnecessary,  because COLLECTOR1 has a localname,  it will automatically write to Collector1Log